components/news/Home.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div class="w-full">
+    <nuxt-link :to="'/news/' + news.slug">
+      <div class="news-image">
+        <img :src="newsImage" :alt="news.title.rendered" class="">
+      </div>
+    </nuxt-link>
+    <div class="grid news-infos">
+      <div class="news-date">
+        <div class="newsgeneral">
+          <nuxt-link to="news/">
+            <h3 class="m-0">
+              News
+            </h3>
+          </nuxt-link>
+        </div>
+        <div class="newsdate">
+          <p>{{ format(new Date(news.modified), 'MM') }}</p>
+          <p>{{ format(new Date(news.modified), 'dd') }}</p>
+        </div>
+      </div>
+      <div class="px-4 news-info-details">
+        <h5 class="py-4">
+          <nuxt-link :to="'/news/' + news.slug">
+            {{ htmlDecoder(news.title.rendered) }}
+          </nuxt-link>
+        </h5>
+        <div v-html="news.excerpt.rendered" />
+        <div class="flex justify-end my-4">
+          <div class="px-4 py-2 bg-gray-200 rounded-lg cursor-pointer more-button" @click="$emit('changenews')">
+            Previous news <i class="fa fa-long-arrow-right" aria-hidden="true" />
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { format } from 'date-fns'
+import { contentApiURL } from '~/constants'
+
+export default {
+  components: {
+  },
+  props: {
+    news: {
+      type: Object,
+      required: true
+    }
+  },
+  data () {
+    return {
+      newsImage: '/img/lahmacun-logo.png',
+      format
+    }
+  },
+  computed: {
+  },
+  watch: {
+    news: {
+      handler () {
+        this.loadNewsImages(this.news.featured_media)
+      }
+
+    }
+  },
+  mounted () {
+    this.loadNewsImages(this.news.featured_media)
+  },
+  methods: {
+    async loadNewsImages (newsId) {
+      const adress = `${contentApiURL}/media/${newsId}`
+      const responseNews = await this.$axios.get(adress)
+      this.newsImage = responseNews.data?.media_details?.sizes?.large?.source_url || responseNews.data?.source_url || this.newsImage
```

Comment:
Cool! This means we can optimise images for example to save bandwith on mobile networks?

Reply:
wp makes different image sizes on upload, thumbnail, medium and large